src/author.rs

```diff
@@ -45,17 +78,72 @@ impl IrohNode {
             Ok(authors)
         })
     }
-}
 
-impl AuthorId {
-    /// Get an [`AuthorId`] from a String.
-    pub fn from_string(str: String) -> Result<Self, IrohError> {
-        let author = iroh::sync::AuthorId::from_str(&str).map_err(IrohError::author)?;
-        Ok(AuthorId(author))
+    /// Export the given author.
+    ///
+    /// Warning: This contains sensitive data.
+    pub fn author_export(&self, author: Arc<AuthorId>) -> Result<Arc<Author>, IrohError> {
+        block_on(&self.rt(), async {
+            let author = self
+                .sync_client
+                .authors
+                .export(author.0)
+                .await
+                .map_err(IrohError::author)?;
+            match author {
+                Some(author) => Ok(Arc::new(Author(author))),
+                None => Err(IrohError::Author {
+                    description: String::from("Author Not Found"),
+                }),
+            }
+        })
     }
 
-    /// Returns true when both AuthorId's have the same value
-    pub fn equal(&self, other: &AuthorId) -> bool {
-        *self == *other
+    /// Import the given author.
+    ///
+    /// Warning: This contains sensitive data.
+    pub fn author_import(&self, author: Arc<Author>) -> Result<Arc<AuthorId>, IrohError> {
+        block_on(&self.rt(), async {
+            self.sync_client
+                .authors
+                .import(author.0.clone())
+                .await
+                .map_err(IrohError::author)?;
+            Ok(Arc::new(AuthorId(author.0.id())))
+        })
+    }
+
+    /// Deletes the given author by id.
+    ///
+    /// Warning: This permanently removes this author.
+    pub fn author_delete(&self, author: Arc<AuthorId>) -> Result<(), IrohError> {
+        block_on(&self.rt(), async {
+            self.sync_client
+                .authors
+                .delete(author.0)
+                .await
+                .map_err(IrohError::author)?;
+            Ok(())
+        })
+    }
+}
+
+mod tests {
+    #[test]
+    fn test_author_api() {
+        let dir = tempfile::tempdir().unwrap();
+        let node = crate::IrohNode::new(dir.into_path().display().to_string()).unwrap();
+
+        let author_id = node.author_create().unwrap();
+        let authors = node.author_list().unwrap();
+        assert_eq!(authors.len(), 1);
+        let author = node.author_export(author_id.clone()).unwrap();
+        assert_eq!(author_id.equal(&author.id()), true);
+        node.author_delete(author_id).unwrap();
+        let authors = node.author_list().unwrap();
+        assert_eq!(authors.len(), 0);
+        node.author_import(author).unwrap();
+        let authors = node.author_list().unwrap();
+        assert_eq!(authors.len(), 1);
     }
 }
```

Comment:
@ramfox if you can review this & write a version of this in python I think we're there